tidb-cloud/changefeed-sink-to-mysql.md

```diff
@@ -78,14 +74,36 @@ The **Sink to MySQL** connector can only sink incremental data from your TiDB cl
 
 After completing the prerequisites, you can sink your data to MySQL.
 
-1. Navigate to the **Changefeed** tab of your TiDB cluster.
+1. Navigate to the **Data Replication** > **Changefeed** tab of your TiDB cluster.
+
 2. Click **Sink to MySQL**.
-3. Fill in the MySQL URL, user, and password.
-    - If you already have data in your TiDB Cluster, you must fill in a specific TSO number that Dumpling provides.
-    - If you do not have any data in your TiDB Cluster, you can choose the "current" TSO.
-4. Click **Test Connectivity**. If your TiDB Cluster can connect to the MySQL service, the **Confirm** button is displayed.
-5. Click **Confirm** and after a while, the sink will begin its work, and the status of the sink will be changed to "**Producing**".
-6. After the operation is complete, set the GC time back (the default value is `10m`):
+
+3. Fill in the MySQL Endpoints, user, and password in **MySQL Connection**.
+
+4. Optionally, if you need to configure the following advanced options, you can click on **Show advanced configuration**.
+    - In **Timezone**, you can select the timezone used to convert time type data, like `timestamp`.
+
+5. Click **Next** to test connectivity
+    - If the connectivity of MySQL is fine, it will go to the next step of configuration.
+    - Otherwise, it will display the connectivity error, which you need to deal with the error before clicking **Next** again.
+
+6. **Table Filter** provides a simple way to filter the tables that need to be replicated by customing [table filter rules](/table-filter.md).
+    - Defaultlly there is a rule `*. *`, which stands for replicating all tables. Whenever you add a new rule, TiDB Cloud will query all the tables in TiDB and display the tables that can be replicated in the list box on the right.  **Note:** The list box on the right does not show the new tables to be replicated in the future, and the schema to be full replicated.
+    - TiDB Cloud also lists tables that do not have unique and primary keys, and these tables may have inconsistent data in downstream during replication due to interrupt recovery. You can choose, not to replicate all tables that do not have unique and primary keys, or replicate these tables.
+
+7. In **Start Position**, configure the starting position for your sink
+    - If you do [Full load data](#full-load-data), you must fill in the TSO that Dumpling provides in **Start replication from a specific TSO**.
+    - If you do not have any data in your TiDB Cluster, you can choose **Start replication from now on**.
+    - Otherwise, you can custom the start time point by choosing **Start replica from a specific time**.
```

Comment:
```suggestion
7. In **Start Position**, configure the starting position for your sink.

    - If you do [full load data](#full-load-data), you must fill in the TSO that Dumpling provides in **Start replication from a specific TSO**.
    - If you do not have any data in your TiDB cluster, you can choose **Start replication from now on**.
    - Otherwise, you can customize the start time point by choosing **Start replication from a specific time**.
```